apps/app.go

```diff
@@ -135,7 +136,7 @@ func marathonAppNameToServiceName(name string, nameSeparator string) string {
 }
 
 func labelsToTags(labels map[string]string) []string {
-	tags := []string{}
+	tags := make([]string, 0, len(labels))
```

Comment:
so why are you benchmarking `BenchmarkApp_RegistrationIntents` when you `not trying to optimize` it? ;)